nautobot_circuit_maintenance/__init__.py

```diff
@@ -90,6 +90,7 @@ class CircuitMaintenanceConfig(PluginConfig):
         "raw_notification_initial_days_since": 7,
         "raw_notification_size": 8192,
         "dashboard_n_days": 30,
+        "overlap_job_log_only_impacts": False,
```

Comment:
we should add this to the Readme to explain it. And, personally, I would prefer a name like "overlap_job_exclude_no_impact_maintenances"